package.json

```diff
@@ -35,7 +35,7 @@
     "@bpmn-io/dmn-migrate": "^0.4.3",
     "@ctrl/ngx-codemirror": "4.0.1",
     "@fortawesome/fontawesome-free": "5.15.1",
-    "@maurodatamapper/mdm-resources": "git+https://github.com/MauroDataMapper/mdm-resources.git#develop",
+    "@maurodatamapper/mdm-resources": "^4.6.0-SNAPSHOT",
```

Comment:
Why has this changed? I thought @olliefreeman said the npm reference to `mdm-resources` should remain pointing at the `develop` branch?

Reply:
yeah agreed i ran a npm updated so not sure why it changed